lib/oli_web/live/collaboration_live/collab_space_config_view.ex

```diff
@@ -72,22 +79,23 @@ defmodule OliWeb.CollaborationLive.CollabSpaceConfigView do
         changeset: changeset,
         page_revision: page_revision,
         page_resource: page_resource,
-        parent_entity: parent_entity
+        parent_entity: parent_entity,
+        topic: topic
       )}
   end
 
   def render(assigns) do
     ~F"""
       <div class="card">
         <div class="card-body d-flex justify-content-between">
-          <div class="d-flex">
-            <div class="card-title h5">Collaborative Space</div>
-            <span class="badge badge-info ml-2" style="height: fit-content">{humanize(@collab_space_status)}</span>
+          <div class="d-flex flex-column">
+            <h3 class="card-title">Collaborative Space Config</h3>
+            <h6 class="d-flex align-items-center">Current status <span class="badge badge-info ml-2" style="height: fit-content">{humanize(@collab_space_status)}</span></h6>
           </div>
 
           {#case @collab_space_status}
             {#match :disabled}
-              <button class="btn btn-outline-primary" :on-click="enable">Enable</button>
+              <div><button class="btn btn-outline-primary" :on-click="enable">Enable</button></div>
```

Comment:
NIT: Seems like an unnecessary div 😄

Reply:
Yes, thank you mate! I saw it and I think it's necessary to wrap the button because if not this happens 👇

![Captura de pantalla 2022-12-22 a la(s) 11 39 04](https://user-images.githubusercontent.com/16328384/209157965-1e411601-271f-4d31-9b31-4c9ccc3c7c24.png)

But if you know how to fix it, I appreciate if you let me know.

Reply:
You're right. Let's leave it then!

Something I noticed is that we are adding divs in every clause of the match instead of just adding one outside the div, something like:

```elixir
<div>
  {#case @collab_space_status}
    {#match :disabled}
      <button class="btn btn-outline-primary" :on-click="enable">Enable</button>
    {#match :enabled}
      <button class="btn btn-outline-primary" :on-click="archive">Archive</button>
      <button class="btn btn-outline-danger" :on-click="disable">Disable</button>
    {#match _}
      <button class="btn btn-outline-primary" :on-click="enable">Enable</button>
      <button class="btn btn-outline-danger" :on-click="disable">Disable</button>
  {/case}
</div>
```

What do you think?

Reply:
Yes! great! It works perfect! I'll fix it!